app/routes/articles/components/ArticleEditor.js

```diff
@@ -121,8 +106,28 @@ const ArticleEditor = ({
   );
 };
 
-export default reduxForm({
+const onSubmit = (
+  data,
+  dispatch,
+  { currentUser, isNew, articleId, submitArticle }: Props
+) => {
+  const body = {
+    ...(isNew ? {} : { id: articleId }),
```

Comment:
When you are already omitting using spread, you might as well do short-circuiting
`...(!isNew && { id: articleId })`

Reply:
:+1:
I just moved the function, so didn't read it.. But yours look way better !